src/controller/index.ts

```diff
@@ -131,6 +134,12 @@ export default class SimulariumController {
                 this.handleTrajectoryInfo(trajFileInfo);
             }
         );
+        this.simulator.setMetricsHandler((metrics: Record<string, unknown>) =>
```

Comment:
can this be typed better? I saw at least you have a Metrics type.  even if that type is a generic Record<string,unknown> it would still be better to use `Metrics` here and above on line 65